mobile-app/app/contracts/TransferDomainV1.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
Could add a section in readme or a comment on where this was taken/generated?

Reply:
Added a [README.md](https://github.com/BirthdayResearch/defichain-wallet/pull/4038/files?short_path=9242be1#diff-9242be105521261c6606959a05d955c59c0a6cd93d166d87256d26e9fa40faf0) inside the `/contracts` folder